config/fcos/v1_5_exp/validate.go

```diff
@@ -39,3 +39,10 @@ func (m BootDeviceMirror) Validate(c path.ContextPath) (r report.Report) {
 	}
 	return
 }
+
+func (e Extension) Validate(c path.ContextPath) (r report.Report) {
+	if e.Name == "" {
+		r.AddOnError(c.Append("extensions"), common.ErrPackageNameRequired)
```

Comment:
The path here is the field name relative to the function receiver:

```suggestion
		r.AddOnError(c.Append("name"), common.ErrPackageNameRequired)
```